### ISSUE TITLE
No longer working

### ISSUE THREAD

**Issue by Th3CracKed** [webdigi/AWS-VPN-Server-Setup#52]
Hi, i was using your vpn for 3 days now but yesterday they stopped working i don't know why.
i tried to reconfigure it multiple time without success

**Comment by webdigi** [comment]
Did you try and reboot instance. 

Alternatively, did the delete stack and create stack work again? What region were you on?

**Comment by Th3CracKed** [comment]
Yeah i tried to reinstall the instance but didn't work, i didn't try to reboot
Region : Paris
Thanks

**Comment by webdigi** [comment]
Thanks so previously it worked for 3 days and not it does not work at all? Can you also please try another region when possible?

Also, please wait for about 10 minutes after first start and try connecting.

**Comment by Th3CracKed** [comment]
Okey i will do that when i have access to my laptop, it works the first
time for 3 days , i did try openvpn beetwen the times and uninstall it
(maybe the cause of this)

**Comment by Th3CracKed** [comment]
Also i did try to wait 10 minutes , same problem

**Comment by Th3CracKed** [comment]
![capture](https://user-images.githubusercontent.com/10481816/39142178-2b03f3b4-472a-11e8-9a04-254ee80399a5.PNG)
same this time i tried usa virginia

**Comment by webdigi** [comment]
have you changed your network in the meantime? Some users had problems with university networks.

Is it possible to try connecting to vpn from a mobile network?

**Comment by Th3CracKed** [comment]
i tried a mobile network got this error : (PPTP) & (L2TP/Ipsec)
![capture](https://user-images.githubusercontent.com/10481816/39143574-923b3836-472e-11e8-8b06-438dde87e1ca.PNG)

**Comment by Th3CracKed** [comment]
the error means : the used port for this connection is closed
